modules/API/pages/built-in-endpoints.adoc

```diff
@@ -754,17 +760,25 @@ If authentication is enabled on RESTPP endpoints, a token needs to be included i
 
 [source,bash]
 ----
-curl -X GET "http://localhost:9000/requesttoken?secret=jiokmfqqfu2f95qs6ug85o89rpkneib3&lifetime=1000000"
+curl -X POST http://localhost:9000/requesttoken \
+     -d '{"secret": "your_secret_value", "lifetime": "100000"}' 
+
+curl --user example_username:example_password \
+     -X GET http://localhost:9000/requesttoken \
+     -d '{"graph": "example_graph"}'
```

Comment:
First of all, I think we can keep using the arbitrary secret `jiokmfqqfu2f95qs6ug85o89rpkneib3` for better understanding, so as the example of the `PUT` and `DELETE` methods.

On the other hand, it would be better if we unify the reference of a keyword in the same context throughout the docs. For example, we use `example_username` here, but some examples in this page use `tigergraph`, and some use `<my_username>`. What do you think @lennessyy, and what would be the best practice in your opinion?

Reply:
> First of all, I think we can keep using the arbitrary secret `jiokmfqqfu2f95qs6ug85o89rpkneib3` for better understanding, so as the example of the `PUT` and `DELETE` methods.
> 
> On the other hand, it would be better if we unify the reference of a keyword in the same context throughout the docs. For example, we use `example_username` here, but some examples in this page use `tigergraph`, and some use `<my_username>`. What do you think @lennessyy, and what would be the best practice in your opinion?

Ah, I realized that my comment didn't send yesterday.

I agree that the arbitrary password is better for helping users understand what a request looks like.

For placeholders, yes it would be nice to have consistency. I've been sticking with `example_something` unless there is a good reason not to. I think we can continue to use `example_something` and I'll formalize it in the style guide.